### PR TITLE
Add happy path tests for internal services

### PR DESCRIPTION
**Pull request by kenoir**
In order to improve upon the interactions of internal service classes we need a foundation of tests across those services.

This PR introduces a bare minimum happy path set of tests.

### CODE REVIEW

File: common/src/test/scala/uk/ac/wellcome/platform/stacks/common/services/CatalogueServiceTest.scala [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+            actualWork shouldBe expectedWork
+          }
+
+        }
+      }
+    }
+
+    describe("getStacksItem") {
+      it("should get a StacksItem for a SierraItemIdentifier") {
```
Comment (jamesgorrie) [inline common/src/test/scala/uk/ac/wellcome/platform/stacks/common/services/CatalogueServiceTest.scala:48]: When do we lookup _from_ SierraIds?

Comment (jamesgorrie) [reply common/src/test/scala/uk/ac/wellcome/platform/stacks/common/services/CatalogueServiceTest.scala:48]: And should the be the duty of the Catalogue service?

Comment (jamesgorrie) [reply common/src/test/scala/uk/ac/wellcome/platform/stacks/common/services/CatalogueServiceTest.scala:48]: Reading through this, I think I might have misunderstood our conversation re: Catalogue vs Sierra vs Stacks.

My understanding was Sierra returns Sierra stuff, catalogue catalogue stuff, and stacks combines the two and returns stacks stuff?

e.g.
```scala
SierraService.getItem("i123456") // only utilises sierra IDs. return SierraItem
CatalogueService.getItem("asdh673j") // only utilised catalogue IDs. return CatalogueItem
StacksService.getItem("asdh673j") // interfaces with catalogue IDs as that's what we expose to clients, but can return both IDs as it knows of both. return StacksItem
```

Comment (kenoir) [reply common/src/test/scala/uk/ac/wellcome/platform/stacks/common/services/CatalogueServiceTest.scala:48]: The Catalogue API contains Sierra Ids & Catalogue Ids for items, and provides the basis for conversion between those identifiers. 

Specifically we look up from Sierra Ids to Catalogue Ids when converting `StacksUserHolds[SierraItemIdentifier]` to  `StacksUserHolds[CatalogueItemIdentifier]`.

TLDR: The catalogue contains sierra ids.

**Review by jamesgorrie — APPROVED** [review]
Tidy!

**Comment by jamesgorrie** [comment]
Merging to use new API shape
